src/data/set6/champions.ts

```diff
@@ -56,14 +56,13 @@ export default {
 	[ChampionKey.Kassadin]: {
 		cast: (elapsedMS, spell, champion) => {
 			const target = champion.target
-			const manaReave = champion.getSpellVariable(SpellKey.ManaReave) ?? 0
-			const castTime = champion.getCurrentSpell()?.castTime ?? 0
-			if (!target) { return console.log('No target', champion.name, champion.team) }
 			champion.queueProjectile(elapsedMS, spell, {
 				onCollision: () => {
```

Comment:
Let's use the parameters of `onCollision: (elapsedMS, affectedUnit) => {` so we can replace `champion.target` with `affectedUnit`, which is guaranteed to be non-null and the champion the Projectile actually collided with.